spec/controllers/achievements_controller_spec.rb

```diff
@@ -130,6 +130,7 @@
         delete :destroy, id: achievement_to_delete.id
 
         expect(response).to have_http_status(:redirect)
+        expect(response).to redirect_to(manage_achievements_path)
```

Comment:
Mmm. I suppose that the assertion on `#132` is redundant in light of the assertion on the next line. Damn.

:worried: :cold_sweat: